http-clients/apache-client/src/test/java/software/amazon/awssdk/http/apache/internal/impl/ApacheHttpRequestFactoryTest.java

```diff
@@ -64,6 +70,28 @@ public void ceateSetsHostHeaderByDefault() {
         assertEquals("localhost:12345", hostHeaders[0].getValue());
     }
 
+    @Test
+    public void requestBase_putRequestWithTransferEncodingChunked_isChunked_isWrapped_doesNotIncludeHeader() {

```

Comment:
nit: can we follow `methodToTest_when_expectedBehavior` (note two `_`s) naming pattern?

Reply:
updated